charts/postgres-cluster/values.yaml

```diff
@@ -38,3 +38,77 @@ global:
     #   cpu: 100m
     #   memory: 128Mi
 
+postgresCluster:
+  instance1:
+    dataVolumeClaimSpec:
+      resources:
+        requests:
+          storage: 10Gi
+
+  patroni:
+    dynamicConfiguration:
+      postgresql:
+        pg_hba:
+          - "host all all 0.0.0.0/0 trust" # This line enable logical replication with programmatic access
+          - "host all postgres 127.0.0.1/32 md5"
+
+  backups:
+    pgbackrest: # Check for more config options https://pgbackrest.org/configuration.html
+      global:
+        repo1-retention-full: "3"  # Keeps 3 full backups
+        repo1-retention-archive: "10"  # Retain the last 10 archive logs
```

Comment:
these are dailys, right? so increase to at least 14 to 2w

is there a MB size cap on these?

Reply:
Good point, thanks! Done!

> frecuency

No, this is weekly, see the schedule conf:
```yaml
full: "0 1 * * 0"  # Weekly full backup every Sunday at 1 AM
differential: "0 3 * * *"  # Daily differential backup
incremental: "*/30 * * * *"  # Every 30 minutes for incremental backups
```

> cap

I've added these, I think it may be want we need here:
```yaml
repo1-bundle-size: "1G"  # Set a 1GB maximum size for each 'backup bundle'
repo1-bundle-limit: "15"    # Limit to 15 'backup bundles'
```